decidim-core/spec/cells/decidim/upload_modal_cell_spec.rb

```diff
@@ -126,4 +126,60 @@ def model_name
       end
     end
   end
+
+  context "when multiple attachments are present" do
```

Comment:
Contexts "when multiple attachments are present" and "when all attachments are images" in this spec come from a different PR: https://github.com/decidim/decidim/pull/10425

Shall we backport them too?

Reply:
Considering that the test suite is Green, and the Tests that are relevant for this PR are a part of the tests added in #10425 I would say that "Yes, backport the tests". 

Also, adding more tests to suite is not affecting the user experience.
Maybe other @decidim/maintainers have other opinion?

Reply:
As far as I know #10425 implemented a fix that did not concern version 0.27.

But if we need that too and if the bug exists in 0.27 too, by all means go ahead with the backport.

For all backports we should aim to follow the original PR that we are backporting. In case the original PR did not have these tests, we should not include them in the backport either unless there is something very relevant regarding this PR in those.

I haven't looked at the actual code that we are backporting here but I'm just giving my point of view on how to handle the backports.